package.json

```diff
@@ -5,7 +5,7 @@
   "main": "./lib/index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "node ./node_modules/gulp/bin/gulp.js build"
+    "build": "gulp build"
```

Comment:
this doesn't mean gulp has to be installed globally.
NPM will use the local version if installed